packages/android_alarm_manager/lib/android_alarm_manager.dart

```diff
@@ -188,7 +218,7 @@ class AndroidAlarmManager {
     assert(callback is Function() || callback is Function(int));
     assert(id.bitLength < 32);
     final int startMillis = time.millisecondsSinceEpoch;
-    final CallbackHandle handle = PluginUtilities.getCallbackHandle(callback);
+    final CallbackHandle handle = InstanceWrappers.getCallbackHandle(callback);
```

Comment:
Is it possible to factor this to an instance method that all it does is to get the `CallbackHandle`. And then in the test, mock the implementation of that instance method? Instead of creating this new static class.

Reply:
Yeah, I can bundle these into `AndroidAlarmManager` itself with static `@visibleForTesting` setters. Initially I avoided that since I thought it would clutter its interface but this isn't great either, that probably makes some more amount of sense.